explanation/dnssec/dnssec.md

```diff
@@ -31,6 +31,55 @@ The public key cryptography underpinning SSL/TLS operates in a similar manner, b
 
 For a more detailed explanation of how the DNSSEC validation is performed, please refer to the [Simplified 12-step DNSSEC validation process](https://bind9.readthedocs.io/en/latest/dnssec-guide.html#the-12-step-dnssec-validation-process-simplified) guide from ISC.
 
+## DNS daemons
+Ubuntu supports multiple DNS resolvers, covering a variety of use cases. Most of them support DNSSEC validation, but it might not be activated and set up with valid trust-anchors automatically.
+
+<!-- Using non-breaking hyphen & non-breaking space for improved table spacing. -->
+| Daemon | Type | DNSSEC support |
+| --- | --- | --- |
+| systemd&#8209;resolved | Stub&nbsp;Resolver&nbsp;(local) | Yes. Disabled by default. Controlled via `DNSSEC=...` setting in `/etc/systemd/resolved.conf.d`. |
+| dnsmasq | Stub Resolver | Yes. Disabled by default. Controlled via `dnssec` and `conf-file=../trust-anchors.conf` settings. |
+| bind9 | Recursive Resolver | Yes. Enabled by default via `dnssec-validation auto;` setting. |
+<!-- TODO: What about "unbound"? -->
+
+The **systemd-resolved** stub resolver is pre-installed in Ubuntu but ships with DNSSEC validation disabled by default. It supports two optional DNSSEC modes: fallback or strict validation. Either mode can be configured using the `DNSSEC=` setting. To do this you need to create a directory for custom `resolved.conf.d` drop-in configuration files, set the corresponding setting inside the `[Resolve]` section and reload the configuration, e.g. in `/etc/systemd/resolved.conf.d/10-dnssec.conf`.
+
+* You can enable the **fallback mode**, using the `DNSSEC=allow-downgrade` setting, which tries to validate the DNSSEC records whenever possible, but at the same time accepts unsigned responses for backwards compatibility with unsigned zones.
+
+* Should authenticity of DNS records be a concern to you, it's advised to opt-in to the **strict mode** by using the `DNSSEC=yes` setting and reloading the configuration:
+
+```
+$ sudo mkdir -p /etc/systemd/resolved.conf.d
+$ sudo tee /etc/systemd/resolved.conf.d/10-dnssec.conf >/dev/null <<EOF
+[Resolve]
+DNSSEC=yes
+EOF
+
+$ sudo systemctl reload systemd-resolved.service
+```
+
+Once reloaded, the functionality of DNSSEC in **systemd-resolved** can be confirmed, using the `dig` command on the local stub resolver at `127.0.0.53:53`, by checking for the existence of the **ad** (Authenticated Data) flag:
```

Comment:
Maybe drop the port, as it's not used in the dig command down below. Just a thought, I'm fine to keep it if you prefer.